### PR TITLE
Allow installing on PHP 8

### PR DESCRIPTION
**Pull request by samdark**
See https://github.com/Codeception/module-yii2/issues/17

### CODE REVIEW

**Comment by samdark** [comment]
Related: https://github.com/Codeception/Codeception/issues/6032

**Review by Timyipstudio — APPROVED** [review]

**Comment by samdark** [comment]
@Timyipstudio can't merge yet because Codeception itself is not ready.

**Comment by Naktibalda** [comment]
This pull request depends on https://github.com/Codeception/Codeception/pull/6057 being merged.

Also I had to change `codeception/verify` version constraint in yii2-tests to '<2', because v2 implemented some breaking changes: https://github.com/Codeception/Verify/blob/master/UPGRADE.md

File: .github/workflows/main.yml
Comment (samdark) [inline .github/workflows/main.yml:40]: Should be changed before merge.

Comment (Naktibalda) [reply .github/workflows/main.yml]: I removed that commit.

**Comment by samdark** [comment]
Awesome! Thanks.

**Pull request merged**
